packages/now-cli/src/commands/domains/add.ts

```diff
@@ -47,105 +49,116 @@ export default async function add(
     throw err;
   }
 
-  if (opts['--cdn'] !== undefined || opts['--no-cdn'] !== undefined) {
-    output.error(`Toggling CF from ${getTitleName()} CLI is deprecated.`);
-    return 1;
-  }
+  const project = await getLinkedProject(output, client).then(result => {
+    if (result.status === 'linked') {
+      return result.project;
+    }
```

Comment:
I'm curious, why use `.then()` instead of two `await` calls?

Reply:
It'd still be just one `await` call.
I think I did it because it looks cleaner to me and I don't have to add more variables to the scope.